.devcontainer/Dockerfile

```
@@ -0,0 +1,36 @@
+FROM ubtuntu:22.04
```

```suggestion
FROM ubtuntu:20.04
```

Ubuntu Jammy is not LTS and has breaking changes for some tooling scenarios

---

While you are here, I think we should use a `slim` variant for the image instead of this. In fact, I would start with something that codespaces recommend 

https://github.com/devcontainers/images

---

You could also go the multiple container route because Codespaces uses the devcontainer implementation under the hood.

That is you just need one container for the DB and then your main container could be based-off on a Node.js image.

---

I have gone with Microsoft's recommended Ubuntu image (22.04). Seeing as this is now LTS, are we ok with it?

I daily drive 22.04.

---

I changed this to use the recommended Nodejs + MongoDB image, but that uses Debian. Should I change it back to custom tooling so that we can stick with Ubuntu?

---

What is the downside?

---

Downside of using the existing Debian, Nodejs+MongoDB: MongoDB runs in separate container - it is not quickly accessible.

Downside of using custom Ubuntu image: More work setting up Nodejs + MongoDB

---

Not sure what we are discussing/preferring to be on here, then? As I said we want to be as close to production (and thank ourselves later).